arm-sql/compositeSql.json

```diff
@@ -4,7 +4,8 @@
     "description": "Composite Swagger for Azure Sql Database Client"
```

Comment:
#Resolved

---
In reply to: [85596076](https://github.com/Azure/azure-rest-api-specs/pull/643#discussion_r85596076) [](ancestors = 85596076)